2/contrib/s2i/run

```diff
@@ -186,6 +186,10 @@ if [[ "$(uname -m)" == "x86_64" ]]; then
 	alternatives --set javac $(alternatives --display javac | awk '/family.*i386/ { print $1; }')
 	export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
     fi
+#set JVM for all other archs
```

Comment:
github review won't let me add a review comment at line 176, but change `multi-arch check ... if not x86_64, leave alone` to `multi-arch check ... if x86_64, pick between 32 and 64 bit`

Reply:
ack. new commit coming soon.